deploy_testnet_example.sh

```diff
@@ -11,10 +11,26 @@ docker run --rm -v "$(pwd)":/code \
 NODE="--node https://rpc.uni.juno.deuslabs.fi:443"
 TXFLAG="--node https://rpc.uni.juno.deuslabs.fi:443 --chain-id uni-3 --gas-prices 0.025ujunox --gas auto --gas-adjustment 1.3 --broadcast-mode block"
 
-# Make sure OWNER and USER wallets have enough JUNOX
-OWNER="$1"
-AGENT="$2"
-USER="$3"
+# Create wallets and make sure they have some JUNOX
+OWNER=owner$RANDOM
+AGENT=agent$RANDOM
+USER=user$RANDOM
```

Comment:
```suggestion
OWNER=croncat-owner-test
AGENT=croncat-agent-test
USER=croncat-user-test
```
and delete keys for those before 
```bash
set -e
```
imo it's better to not fill keyring with test keypairs, what you think? @NinoLipartiia